src/components/FollowersFollowing.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { Button } from "antd";
+import FollowersFollowingUsers from "./FollowersFollowingUsers";
+import { preFabProfiles } from "../test/testProfiles";
+
+const FollowersFollowing = (): JSX.Element => {
```

Comment:
This name is a bit confusing. My first though is that this was a component that was monitoring the followers that are following. And made me wonder where the FolloweesFollowing component was.

I think a name like `ProfileFollows` would be clearer. It's the component that handles all follows for the profile.